### ISSUE TITLE
Datepicker input field with prepend symbol

### ISSUE THREAD

**Issue by rogeriopvl** [sapo/Ink#424]
I have an input field that is used for a Datepicker. I'm trying to prepend a symbol to the input field, but the Datepicker code creates a wrapper div around it, invalidating the CSS selector for the `append-symbol` class.

Is there a way around this? Or maybe I'm missing something?

Here's my markup:

```
<div class="ink-form">
  <div class="control-group column-group no-margin">
    <div class="control all-30 small-100 tiny-100 append-symbol">
      <span>
        <input type="text" id="myDatePicker" class="ink-datepicker">
        <i class="fa fa-calendar"></i>
      </span>
    </div>
  </div>
</div>
```

Thanks!

**Comment by mariomc** [comment]
Just add a "control" class to the direct parent of the input.
Datepicker won't add a wrapper if the parent of the input matches the selector ".ink-form .control-group .control". We probably should add ".append-symbol > span, .preprend-symbol > span" to that same selector.

Check the example here: http://jsbin.com/cegudo/5/edit

**Comment by rogeriopvl** [comment]
I just confirmed that this issue is fixed in 3.1.4. The `span.control` workaround is no longer needed.

I'm going to close it.

Thanks :+1:
